### PR TITLE
Add "Player color theme" setting

### PR DESCRIPTION
**Pull request by nt4f04uNd**
Add a setting to choose how to color player interface.
Now interface is by default colored based on the current song art.

<details>
<summary>Video</summary>

https://user-images.githubusercontent.com/39104740/193460408-41f0ade3-f9c3-4244-a5ce-2e2885032c2f.mp4
</details>

### CODE REVIEW

File: lib/localization/arb/intl_de.arb [hunk 1]
```diff
@@ -310,5 +310,8 @@
   "accept": "Akzeptieren",
   "cancel": "Abbrechen",
   "copied": "Kopiert",
-  "gitHubRepo": "GitHub Repository"
+  "gitHubRepo": "GitHub Repository",
+  "defaultValue": "Standard",
+  "playerInterfaceColorStyle": "Player-Farbthema",
+  "playerInterfaceColorStyleArtColor": "Basierend auf Track-Art"
```
Comment (nt4f04uNd) [inline lib/localization/arb/intl_de.arb:316]: @Abestanis please take a look whether Google translate pulled off a good German localization :-)

File: lib/widgets/show_functions.dart [hunk 2]
```diff
@@ -245,4 +245,52 @@ class ShowFunctions extends NFShowFunctions {
...
+            },
+          ),
+        ),
+      );
+    }
+
+    return ShowFunctions.instance.showAlert(
+      context,
+      ui: theme.systemUiThemeExtension.modalOverGrey,
+      title: Text(l10n.sort),
```
Comment (Abestanis) [inline lib/widgets/show_functions.dart:286]:
```suggestion
      title: Text(title),
```
I noticed this in the video.

**Review by Abestanis — COMMENTED** [review]
This is an awesome feature, really cool. Thanks for the video showcasing it!
I don't have much time right now, but I quickly looked over the code and only found a small copy and paste mistake, the rest looks good👍.

I think having a `System` choice which could use the color from the Material You on Android and the accent color on Windows would be a cool addition on top of this, once this is merged. I'll try this, once I have more time.

File: lib/localization/arb/intl_de.arb [hunk 3]
```diff
@@ -310,5 +310,9 @@
   "accept": "Akzeptieren",
   "cancel": "Abbrechen",
   "copied": "Kopiert",
-  "gitHubRepo": "GitHub Repository"
+  "gitHubRepo": "GitHub Repository",
+  "defaultValue": "Standard",
+  "playerInterfaceColorStyle": "Player-Farbthema",
+  "playerInterfaceColorStyleArtColor": "Basierend auf dem Coverbild",
+  "playerInterfaceColorStyleThemeBackgroundColor": "Hintergrundfarbe des Farbthema"
```
Comment (Abestanis) [inline lib/localization/arb/intl_de.arb:317]:
```suggestion
  "playerInterfaceColorStyleThemeBackgroundColor": "Hintergrundfarbe des Farbthemas"
```

Comment (nt4f04uNd) [reply lib/localization/arb/intl_de.arb]: Thanks :)

**Comment by nt4f04uNd** [comment]
Phew, finally, merged this!
Been using it for a few months now, only noticed small bugs and fixed them.
Mostly it took me a while to figure out how to make this whole thing work with tests.

**Pull request merged**